src/clang_parser.cpp

```diff
@@ -663,8 +666,8 @@ bool ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
     {
       if (get_unknown_type(msg) != "" && !bpftrace.force_btf_)
```

Comment:
I think we can remove `bpftrace.force_btf_`.

Reply:
Ah, I overlooked this one. Thanks